django/core/signing.py

```diff
@@ -151,6 +155,14 @@ class Signer(object):
     def __init__(self, key=None, sep=':', salt=None):
         # Use of native strings in all versions of Python
         self.sep = force_str(sep)
+        sep_is_invalid = sep != ":" and (
+            not self.sep or
+            self.sep[0] in _SEP_UNSAFE or
```

Comment:
why checking only the first and last values of `self.sep`? how about `any(sep in _SEP_UNSAFE for sep in self.sep)`?